src/Bundle/BundleExtension.php

```diff
@@ -2,71 +2,88 @@
 
 namespace Becklyn\RadBundle\Bundle;
 
+use Becklyn\RadBundles\Bundle\BundleExtension as NewBundleExtension;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\Container;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\Extension;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\HttpKernel\Bundle\BundleInterface;
 
-/**
- * Base class to use in your bundle to easily create an extension.
- */
-class BundleExtension extends Extension
+if (\class_exists(NewBundleExtension::class))
```

Comment:
You can remove the `class_exists()`, because the class will always exist? We load it via Composer.